### ISSUE TITLE
Webinar: Automated Accessibility Testing with WorldSpace for iOS 

### ISSUE THREAD

**Issue by ljgoslin** [a11yproject/a11yproject.com#571]
On Tuesday, January 23rd at 2:00PM EST, join our live webinar hosted by Deque's mobile accessibility expert, Chris McMeeking. Chris will teach developers how they can now trial WorldSpace for iOS for free and utilize the limited functionality framework in their own build.

By downloading this free trial, developers will be able to:
- Run WorldSpace for iOS on their application.
- View WCAG 2.0 AA violations in the interactive automated testing framework.
- Learn accessibility best practices from the Deque iOS pattern library rule sets examples.

Register at this **[link]**(https://accessibility.deque.com/live-webinar-native-mobile-ios-accessibility-testing).

**Comment by ericwbailey** [comment]
Addd in https://github.com/a11yproject/a11yproject.com/pull/573.
